cartographer/mapping/map_builder.cc

```diff
@@ -303,10 +309,11 @@ std::map<int, int> MapBuilder::LoadState(
         proto.mutable_submap()->mutable_submap_id()->set_trajectory_id(
             trajectory_remapping.at(
                 proto.submap().submap_id().trajectory_id()));
-        submap_id_to_submap.Insert(
+        const auto submap_id =
```

Comment:
nit: Isn't it more readable to write `const SubmapId submap_id( ... )` similar to line 322?